internal/k8shandler/collection.go

```diff
@@ -112,13 +112,21 @@ func (clusterRequest *ClusterLoggingRequest) CreateOrUpdateCollection() (err err
 	var httpInputs []string
 	for _, input := range clusterRequest.Forwarder.Spec.Inputs {
 		if input.Receiver != nil && input.Receiver.HTTP != nil {
-			httpInputs = append(httpInputs, input.Name)
+			serviceName := fmt.Sprintf("%s-%s-%s", input.Name, clusterRequest.Forwarder.Namespace, clusterRequest.Forwarder.Name)
```

Comment:
we should bury the servicename or the base of the service name, or even a function on that struct that creates the servicename in ResourceNames